CHANGELOG.md

```diff
@@ -8,13 +8,14 @@ All notable changes to the Pony compiler and standard library will be documented
 
 - Avoid fairly easy to trigger overflow in Windows Time.nanos code ([PR #4227](https://github.com/ponylang/ponyc/pull/4227))
 - Fix incorrect interaction between String/Array reserve and Pointer realloc ([PR #4223](https://github.com/ponylang/ponyc/pull/4223))
+- Fix broken docgen on Windows ([PR #4226](https://github.com/ponylang/ponyc/pull/4223))
```

Comment:
For some reason, I can't suggest a change anymore. No idea why. The URL points to the wrong PR.

Reply:
Instead of docgen, can we say "documentation generation"